homeassistant/components/overseerr/const.py

```diff
@@ -2,7 +2,44 @@
 
 import logging
 
+from python_overseerr.models import NotificationType
+
 DOMAIN = "overseerr"
 LOGGER = logging.getLogger(__package__)
 
 REQUESTS = "requests"
+
+REGISTERED_NOTIFICATIONS = (
+    NotificationType.REQUEST_PENDING_APPROVAL
+    | NotificationType.REQUEST_APPROVED
+    | NotificationType.REQUEST_DECLINED
+    | NotificationType.REQUEST_AVAILABLE
+    | NotificationType.REQUEST_PROCESSING_FAILED
+    | NotificationType.REQUEST_AUTOMATICALLY_APPROVED
+)
+JSON_PAYLOAD = (
```

Comment:
how HA specific is this/could this be generated in the lib based on notification types as input (tbh I don't really love having this here)

Reply:
![image](https://github.com/user-attachments/assets/1670d7f6-7a1b-4762-9de9-c62a02c6cf44)

So Overseerr has the option to shape the webhook however you want. And I don't really think that this is part of the library, as the library does not consume the webhook but HA does

Reply:
I get that, I was thinking about moving that shaping part to the lib, just so we don’t handle raw json here. Can we at least prettify the formatting (triple string) if it stays here for readability?

Reply:
The fun part is that I removed all space in between, as apparently overseerr seems to mess with spaces.

But I think moving the shape to the library leaves you still with a mess because how are you going to shape parameters for that

Reply:
was thinking about passing lists per top level key since it's always `key: {{key}}`

Reply:
It is not

![image](https://github.com/user-attachments/assets/b4ef4df0-a54e-46fb-b0a0-84c5adf54fc8)

Reply:
But someone can wish to have a different layout and not have these keys there

Reply:
I get that, was just looking for a way to avoid a large raw json block here